spec/factories/submission.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :submission do
+    submission_type 0
```

Comment:
This is what a tutorial on booleans in migrations told me - I thought it is not possible to use model-scoped literals here. Will try.